### PR TITLE
chore: bump to 7.19.2

### PR DESCRIPTION
**Pull request by lesterchoi-okta**
## Description:

bumps version to 7.19.2, updates SRI hashes

## PR Checklist

- [ ] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-736500](https://oktainc.atlassian.net/browse/OKTA-736500)

### CODE REVIEW

File: polyfill/README.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ To embed the Sign-in Widget via CDN, include links to the JS and CSS files in yo
...
 
 <!-- Widget bundle for Okta Identity Engine -->
-<script src="https://global.oktacdn.com/okta-signin-widget/7.19.1/js/okta-sign-in.oie.min.js" type="text/javascript" integrity="sha384-dsJo3gNsmsiwWA84RCi8/8QDZuUuO7qakY04G8EpErXcQStprNhpFUNUOo9OQcLw" crossorigin="anonymous"></script>
+<script src="https://global.oktacdn.com/okta-signin-widget/7.19.2/js/okta-sign-in.oie.min.js" type="text/javascript" integrity="sha384-nnD0YVNPUCj8Y/E/9yQaDjRZCHz8D3sDi87m5TkMb0P2HLvjNgnqrYC+uFzq7DVd" crossorigin="anonymous"></script>
 
 <!-- CSS for widget -->
-<link href="https://global.oktacdn.com/okta-signin-widget/7.19.1/css/okta-sign-in.min.css" type="text/css" rel="stylesheet" integrity="sha384-FL6VsXYuwuq1Zo5lnWVgcOMWSo1JuvDue2bQ66/TdWOTwPEU9OgvF4Ks8fNnaiHd" crossorigin="anonymous" />
+<link href="https://global.oktacdn.com/okta-signin-widget/7.19.2/css/okta-sign-in.min.css" type="text/css" rel="stylesheet" integrity="sha384-4NICyd+flTmlD7KwjKUhd8sZF67pwe/qjqM7A97rp5Lay0KApG081cyie7+5+w0M" crossorigin="anonymous" />
```
Comment (lesterchoi-okta) [inline polyfill/README.md:18]: @denysoblohin-okta are we expecting a change to the `okta-sign-in.min.css` bundle?

File: polyfill/README.md [hunk 2]
```diff
@@ -9,13 +9,13 @@ To embed the Sign-in Widget via CDN, include links to the JS and CSS files in yo
 
 ```html
 <!-- Polyfill for older browsers -->
-<script src="https://global.oktacdn.com/okta-signin-widget/7.19.1/js/okta-sign-in.polyfill.min.js" type="text/javascript" integrity="sha384-QzQIGwIndxyBdHRQOwgjmQJLod6LRMchZyYg7RUq8FUECvPvreqauQhkU2FF9EGD" crossorigin="anonymous"></script>
+<script src="https://global.oktacdn.com/okta-signin-widget/7.19.2/js/okta-sign-in.polyfill.min.js" type="text/javascript" integrity="sha384-QzQIGwIndxyBdHRQOwgjmQJLod6LRMchZyYg7RUq8FUECvPvreqauQhkU2FF9EGD" crossorigin="anonymous"></script>
```
Comment (lesterchoi-okta) [inline polyfill/README.md:12]: @denysoblohin-okta `okta-sign-in.polyfill.min.js` doesn't change?

Comment (lesterchoi-okta) [reply polyfill/README.md:12]: nvm. the dompurify downgrade should only affect G3 bundle

File: polyfill/README.md (hunk 1 above)
Comment (trevoring-okta) [reply polyfill/README.md:18]: Doesn't feel like this should change?

File: README.md [hunk 3]
```diff
@@ -269,13 +269,13 @@ When using one of the bundles without the polyfill included, you may want to con
...
 
 <!-- Widget bundle for Okta Identity Engine -->
-<script src="https://global.oktacdn.com/okta-signin-widget/7.19.1/js/okta-sign-in.oie.min.js" type="text/javascript" integrity="sha384-dsJo3gNsmsiwWA84RCi8/8QDZuUuO7qakY04G8EpErXcQStprNhpFUNUOo9OQcLw" crossorigin="anonymous"></script>
+<script src="https://global.oktacdn.com/okta-signin-widget/7.19.2/js/okta-sign-in.oie.min.js" type="text/javascript" integrity="sha384-nnD0YVNPUCj8Y/E/9yQaDjRZCHz8D3sDi87m5TkMb0P2HLvjNgnqrYC+uFzq7DVd" crossorigin="anonymous"></script>
 
 <!-- CSS for widget -->
-<link href="https://global.oktacdn.com/okta-signin-widget/7.19.1/css/okta-sign-in.min.css" type="text/css" rel="stylesheet" integrity="sha384-FL6VsXYuwuq1Zo5lnWVgcOMWSo1JuvDue2bQ66/TdWOTwPEU9OgvF4Ks8fNnaiHd" crossorigin="anonymous" />
+<link href="https://global.oktacdn.com/okta-signin-widget/7.19.2/css/okta-sign-in.min.css" type="text/css" rel="stylesheet" integrity="sha384-FL6VsXYuwuq1Zo5lnWVgcOMWSo1JuvDue2bQ66/TdWOTwPEU9OgvF4Ks8fNnaiHd" crossorigin="anonymous" />
```
Comment (lesterchoi-okta) [inline README.md:278]: rebased and removed css font change

**Review by trevoring-okta — APPROVED** [review]

**Pull request merged**